### PR TITLE
Fix UI not loading after merge conflict issue

### PR DESCRIPTION
**Pull request by nvazquez**
### Description

This PR fixes the UI not loading due to unresolved merge conflict

image

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Review by rohityadavcloud — APPROVED** [review]

**Review by weizhouapache — APPROVED** [review]
tested ok

**Comment by nvazquez** [comment]
@blueorangutan ui

**Comment by rohityadavcloud** [comment]
Tested locally - OK

**Review by shwstppr — APPROVED** [review]

File: ui/public/locales/en.json [hunk 1]
```diff
@@ -668,15 +668,12 @@
 "label.duration.7days": "7 days",
 "label.dynamicscalingenabled": "Dynamic scaling enabled",
 "label.dynamicscalingenabled.tooltip": "VM can dynamically scale only when dynamic scaling is enabled on template, service offering and global setting.",
-<<<<<<< HEAD
 "label.diskofferingstrictness": "Disk offering strictness",
```
Comment (harikrishna-patnala) [inline ui/public/locales/en.json:671]: Thanks for the PR @nvazquez . There are duplicates also, can you please check them too.

**Review by DaanHoogland — APPROVED** [review]
cltgm

**Comment by nvazquez** [comment]
@blueorangutan ui

File: ui/public/locales/en.json (hunk 1 above)
Comment (rohityadavcloud) [reply ui/public/locales/en.json:671]: @harikrishna-patnala this is the only place I can see the `HEAD`; do you see other places in the UI or backend code with merge conflicts?

**Pull request merged**